tests/test_flatten.py

```diff
@@ -8,18 +13,29 @@
         "contextInfo": {"context1": "info1", "context2": "info2"},
     }
 ]
+PD_DATA = DataFrame(DICT_DATA)
+GEOMETRY = [Point(4, 2)]
+GEOPD_DATA = GeoDataFrame(DICT_DATA, crs="EPSG:4326", geometry=GEOMETRY)
 
 
 def teste_flatten_wrong_nested_columns_value_error():
```

Comment:
As far as I know `pytest` runs functions starting with `test_` (not `teste_`). Can you confirm this test is running?